integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -211,13 +211,16 @@ def get_params(init_list, marked_params=[]):
 @incompat
 @pytest.mark.parametrize('data_gen', _init_list_no_nans_with_decimal, ids=idfn)
 @pytest.mark.parametrize('conf', get_params(_confs, params_markers_for_confs), ids=idfn)
-def test_hash_grpby_sum(data_gen, conf):
+def test_hash_grpby_sum(data_gen, conf, rapids_shuffle_enabled):
```

Comment:
I'm not a fan of the "magical" behavior for `rapids_shuffle_enabled`.  Why not treat this like any other config setting parameter, like is done with the max batch size setting in some of the other tests?

Reply:
My goal was to have a few select tests that would toggle the shuffle manage on, with all the other tests having it off. Currently that's not actually happening, so I'll need to fix that, but this case different than other parameters. Perhaps I should just duplicate the test case with the shuffle enabled config set to true, and put a marker around that.

If the shuffle manager isn't configured, the user shouldn't pass --rapids_shuffle_enabled, and this test would execute half the cases. So this is an optional parametrization, and I haven't found a good way to do this in pytest.

Reply:
Ok thinking about this more, I am not 100% sure how this will be used in the tests.

We may need to run a separate pytest job (with standalone mode), or we may need to only run this in certain environments. Given the uncertainty with this I removed the changes in the integration test until that is well defined.